app/controllers/reports/regions_controller.rb

```diff
@@ -141,14 +155,25 @@ def find_region
     end
   end
 
+  def find_region_v2
+    region_type = report_params[:report_scope]
+    @region = if region_type == "facility_district"
+      FacilityDistrict.new(name: report_params[:id], scope: current_admin.accessible_facilities(:view_reports))
```

Comment:
@harimohanraj89 can you remind me what the long term plan for "FacilityDistrict" is? Can this go away once we have regions fully in placed and production data cleaned up?

Reply:
Totes. Our goal should still be to axe this thing asap.

I chatted with Vinay. Since the app can intelligently purge data outside a user's sync group regularly now, we should be able to rearrange facilities and groups more freely now. The rough order of operations should be:
* Launch block-level sync
* Test in SBX what happens if we split facilities in one facility group into two groups
* Test in SBX what happens if we combine facilities from multiply facility groups into one group
* If it looks good, make the data changes in prod
  * Split Bathinda and Mansa into two FGs
  * Combine Chennai into a single FG
* Axe the `FacilityDistrict`